tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/CustomJspPage.java

```diff
@@ -1039,8 +1042,44 @@ private IRuntime getLiferay70Runtime()
         }
         else
         {
-            return null;
+            IRuntime liferay70Runtime = getLiferay70ExistServerRuntime();
+
+            if( liferay70Runtime != null )
+            {
+                return liferay70Runtime;
+            }
+            else
+            {
+                Boolean openAddLiferaryServerDialog = MessageDialog.openQuestion(
+                    UIUtil.getActiveShell(), "Could not convert",
+                    "Could not find Liferay 7.x Server, do you want to add one?" );
+
+                if( openAddLiferaryServerDialog )
+                {
+                    ServerUIUtil.showNewServerWizard( UIUtil.getActiveShell(), "liferay.bundle", null, "com.liferay." );
+                    return getLiferay70ExistServerRuntime();
+                }
+            }
         }
+        return null;
+    }
+
+    private IRuntime getLiferay70ExistServerRuntime()
+    {
+        IServer[] servers = ServerCore.getServers();
+
+        if( !CoreUtil.isNullOrEmpty( servers ) )
+        {
+            for( IServer existServer : servers )
+            {
+                if( LiferayServerCore.newPortalBundle( existServer.getRuntime().getLocation() ) != null )
```

Comment:
Instead of creating a new portal bundle you should just check the runtime type of the server that you found.  If the runtimeType is the same as Liferay 7.x server then we can use that runtime.